lib/igniter/phoenix/generator.ex

```diff
@@ -0,0 +1,117 @@
+defmodule Igniter.Phoenix.Generator do
+  @moduledoc false
+  # Wrap Phx.New.Generator
+  # https://github.com/phoenixframework/phoenix/blob/7586cbee9e37afbe0b3cdbd560b9e6aa60d32bf6/installer/lib/phx_new/generator.ex#L69
+
+  def copy_from(igniter, project, mod, name) when is_atom(name) do
+    mapping = mod.template_files(name)
+
+    templates =
+      for {format, _project_location, files} <- mapping,
+          {source, target_path} <- files,
+          source = to_string(source) do
+        # target = Phx.New.Project.join_path(project, project_location, target_path)
+        target = expand_path_with_bindings(target_path, project)
+        {format, source, target}
+      end
+
+    Enum.reduce(templates, igniter, fn {format, source, target}, acc ->
+      case format do
+        :keep ->
+          acc
+
+        :text ->
+          contents = mod.render(name, source, project.binding)
+          Igniter.create_new_file(acc, target, contents, on_exists: :overwrite)
```

Comment:
`:overwrite` here has the same behavior as calling `https://hexdocs.pm/mix/Mix.Generator.html#create_file/3` with `:force`, currently I think this is the only way to make it work but I believe we can improve it with https://github.com/ash-project/igniter/issues/127

Reply:
We *could* prompt the user if we wanted, but I think showing the diff is enough.